src/components/autocomplete/Autocomplete.vue

```diff
@@ -78,9 +85,16 @@
                         class="dropdown-item is-disabled">
                         <slot name="empty" />
                     </div>
-                    <div v-if="hasFooterSlot" class="dropdown-item">
+                    <a
+                        v-if="hasFooterSlot"
+                        class="dropdown-item"
+                        role="button"
+                        tabindex="0"
+                        :class="{ 'is-hovered': 'footer' === hovered }"
```

Comment:
I am not sure about it.. is it a good to show the item with with hovered class? I should try it before..

Reply:
I think it needs the hover effect otherwise it's very hard to tell when you have the header or footer selected.

Reply:
To clarify I made it so that it only shows the hover when using the arrow keys. It won't show when hovered by a mouse.